frame/staking/src/lib.rs

```diff
@@ -352,6 +358,9 @@ type NegativeImbalanceOf<T> = <<T as Config>::Currency as Currency<
 	<T as frame_system::Config>::AccountId,
 >>::NegativeImbalance;
 
+type AccountIdOf<T> = <T as frame_system::Config>::AccountId;
```

Comment:
So far I've found this to be a bit superfluous. typing `T::AccountId` is literally less keystroked than `AccountIdOf<T>`

Reply:
That's true for all cases where `T::AccountId` works, but occasionally, the type system decides that it needs the more explicit form. Once the type definnition existed, it seemed simpler to just use it instead of checking each instance to see whether rust would accept the other form.